.github/workflows/code-checks.yml

```diff
@@ -124,7 +124,7 @@ jobs:
       run: |
         cd asv_bench
         asv machine --yes
-        asv run --quick --dry-run --durations=30 --python=same
+        asv run --quick --dry-run --durations=30 --python=same --show-stderr
```

Comment:
is this something we want to keep or just for debugging this PR?

Reply:
I think it'd be useful to keep. We run the ASVs on the CI just to see if they error out, so it seems to make sense to also say what the error was.